packages/@aws-cdk/core/test/test.stack.ts

```diff
@@ -654,6 +654,41 @@ export = {
     test.done();
   },
 
+  'feature flag aws:cdk:enable-stack-name-duplicates will use the artifact id as the template name'(test: Test) {
+    // GIVEN
+    const app = new App({
+      context: {
+        [cxapi.ENABLE_STACK_NAME_DUPLICATES_CONTEXT]: 'true'
```

Comment:
Should just be the boolean `true`, not the string `'true'`, right?

Reply:
context values are always strings